README.rst

```diff
@@ -0,0 +1,75 @@
+Linux Integration Services Automation (LISA)
+============================================
+
+|CI Workflow| |GitHub license|
+
+**Linux Integration Services Automation (LISA)** is a Linux quality
+validation system, which consists of two parts：
+
+-  A test framework to drive test execution.
+-  A set of test suites to verify Linux kernel/distribution quality.
+
+``LISA`` was originally designed and implemented for Microsoft Azure and
+Windows HyperV platforms; now it can be used to validate Linux quality
+on any platforms if the proper orchestrator module implemented.
+
+Why LISA
+--------
+
+-  **Scalable**\ ：Benefit from the appropriate abstractions, ``LISA``
+   can be used to test the quality of numerous Linux distributions
+   without duplication of code implementation.
+
+-  **Customizable**: The test suites created on top of ``LISA`` can be
+   customized to support different quality validation needs.
+
+-  **Support multiple platforms**: ``LISA`` is created with modular
+   design, to support various of Linux platforms including Microsoft
+   Azure, Windows HyperV, Linux bare metal, and other cloud based
+   platforms.
+
+-  **End-to-end**: ``LISA`` supports platform specific orchestrator to
+   create and delete test environment automatically; it also provides
+   flexibility to preserve environment for troubleshooting if test
+   failed.
+
+Documents
+---------
+
+-  `Quick start <https://linux-integration-services-automation-lisa.readthedocs.io/en/latest/quick_start.html>`__
+-  `Run tests <https://linux-integration-services-automation-lisa.readthedocs.io/en/latest/docs/run_test/run.html>`__
```

Comment:
The url is too long to remember. Can we have a shorter one like `https://lisa.readthedocs.io` or ``https://mslisa.readthedocs.io` if there is conflict?

Reply:
Why not MS's https://microsoft.github.io/?

Reply:
@squirrelsc Yes, I can change it (technically, build a new one) to mslisa.
@glima Could you be more specific on that?

Reply:
Well, Github has its own doc/webpage hosting system, why use a third party one? Just at what Andy did: https://microsoft.github.io/lisa/

Reply:
It's a good choice. The key difference is the github.io doesn't save pages. The static pages must be submitted to some repo. It's not friendly for hosting multiple version of documents. Either allow a workflow to commit code in CI, or use other repos or branches to save the generated pages. LISA will support versions in future, it means every version release need a separated page cache.